0/mdav2/geom.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import masses
+
+class Molecule:
+    """
+    defines the Molecule class. Molecule objects are initialized with a .xyz
+    filename as input. 
+    """
+    def __init__(self,filename):
+        self.moleculefile = open(filename,'r')
+        self.moleculelines = self.moleculefile.readlines()
+        
+        #units are supplied on line 2 
+        self.units = str(self.moleculelines[1]).rstrip()
+        #number of atoms is supplied on line 1   
+        self.natom = int((self.moleculelines[0]).rstrip())
+        
+        #create empty lists of atom labels, masses, charge, and geometry matrix         for later population
+        self.labels = [0]*self.natom
+        self.masses = [int(0)]*self.natom
+        self.charges = [float(0)]*self.natom
+        self.geom = np.zeros(shape=(self.natom,3))
```

Comment:
Nice use of the numpy functions! You can simplify this to `np.zeros((self.natom, 3))`, as shape is the first argument passed to np.zeros.
```
Docstring:
zeros(shape, dtype=float, order='C')
```